src/components/list/SearchList.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Ref, forwardRef } from "react";
+import classNames from "classnames";
+import { Button, LoadingSpinner } from "hds-react";
+
+import styles from "./searchList.module.scss";
+import Text from "../text/Text";
+
+type Props = {
+  items: React.ReactElement[];
+  onLoadMore: () => void;
+  hasNext: boolean;
+  loading: boolean;
+  count: number;
+  blockSize: number;
+};
+
+const SearchList = forwardRef(
+  (
+    { loading, hasNext, count, onLoadMore, items, blockSize }: Props,
+    ref: Ref<HTMLLIElement>
+  ) => {
+    const resultsLeft = count ? count - items.length : 0;
+    const a11yIndex = (Math.floor(items.length / blockSize) - 1) * blockSize;
+    const loadedMoreAmount = hasNext ? blockSize : count - items.length;
+
+    return (
+      <>
+        {!loading && (
+          <Text variant="h2" className={styles.resultCount} role="status">
+            {items.length} hakutulosta
```

Comment:
I think we would rather want to show the total result count here?

Reply:
That would indeed make more sense.